examples/offloading-with-policies/manifests/testing-deploy.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pod-south
```

Comment:
Should be probably modified to `app-south`, which is more generic (as this is no longer a single pod). Same for all occurrences.